multicluster/controllers/multicluster/member/clusterset_controller.go

```diff
@@ -109,6 +109,12 @@ func (r *MemberClusterSetReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			r.clusterID = common.InvalidClusterID
 			r.clusterSetID = common.InvalidClusterSetID
 		}
+		if r.remoteCommonArea == nil {
+			klog.InfoS("Clean up all resources created by Antrea Multi-cluster controller when the ClusterSet is deleted", "clusterset", req.NamespacedName)
```

Comment:
How do we remove `MemberClusterAnnounce` in the leader?

Reply:
When there is a ClusterSet CR deletion event, the corresponding `MemberClusterAnnounce` will be deleted as well.  This is done by `r.deleteMemberClusterAnnounce(ctx)` on the line 101.
If there is a case that member MC controller is crashed during ClusterSet deletion, Leader's stale controller will check the update timestamp annotation to delete it when it's no longer updated after 24 hours.

Reply:
Ok.